src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -140,8 +141,7 @@ class CollapsibleCourseRun extends React.Component {
           type="date"
           component={RenderInputTextField}
           format={value => getDateString(value)}
-          normalize={value => moment.utc(value)
-          .toISOString()}
+          normalize={value => moment.utc(value).toISOString()}
```

Comment:
If you don't mind, update the normalize for the go_live_date to also be one line.